appserver/concurrent/concurrent-impl/src/main/java/org/glassfish/concurrent/admin/ManagedExecutorServiceBaseManager.java

```diff
@@ -154,10 +154,12 @@ protected ResourceStatus isValid(Resources resources, boolean validateResourceRe
         }
         status = resourcesHelper.validateBindableResourceForDuplicates(resources, jndiName, validateResourceRef, target, clazz);
 
-        try {
-            Integer.parseInt(corePoolSize);
-        } catch (NumberFormatException nfe) {
-            return new ResourceStatus(ResourceStatus.FAILURE, localStrings.getLocalString("coresize.must.be.number", "Option corepoolsize must be a number."));
+        if ("false".equals(useForkJoinPool)) {
+            try {
+                Integer.parseInt(corePoolSize);
+            } catch (NumberFormatException nfe) {
+                return new ResourceStatus(ResourceStatus.FAILURE, localStrings.getLocalString("coresize.must.be.number", "Option corepoolsize must be a number."));
+            }
```

Comment:
Knock-on from suggestion to move `ManagedExecutorService` specific config out of `ManagedExecutorServiceBase` interface, this would need adjusting.

Reply:
done